README.md

```diff
@@ -1 +1,127 @@
-# atlas-cli
\ No newline at end of file
+# Atlas CLI
+This command-line tool helps developers become productive on Atlas. It aims to provide a better development experience by reducing the initial time and effort it takes to build applications.
+
+## Getting Started
+These instructions will help you get the Atlas command-line tool up and running on your machine.
+
+### Prerequisites
+Please install the following dependencies before running the Atlas command-line tool.
+
+#### goimports
+
+The `goimports` package resolves missing import paths in Go projects. It is part of the [official Go ecosystem](https://golang.org/pkg/#other) and can be installed with the following command:
+```
+$ go get golang.org/x/tools/cmd/goimports
+```
+#### dep
+
+This is a dependency management tool for Go. You can install `dep` with Homebrew:
+
+```sh
+$ brew install dep
+```
+More detailed installation instructions are available on the [GitHub repository](https://github.com/golang/dep).
+
+#### golang-migrate
+Bootstrapped applications use [golang-migrate](https://github.com/golang-migrate/migrate) for database migrations. You can install the `migrate` binary with the standard Go toolchain:
+
+```
+$ go get -u -d github.com/golang-migrate/migrate/cli github.com/lib/pq
+$ go build -tags 'postgres' -o /usr/local/bin/migrate github.com/golang-migrate/migrate/cli
```

Comment:
Running this command gives me 
```
go build github.com/golang-migrate/migrate/database/postgres: no buildable Go source files in /Users/chorst/git/src/github.com/golang-migrate/migrate/database/postgres
```
The same approach worked for Mattes for me though. Is there a different installation command for golang-migrate, that the docs haven't been updated with yet that we should include here?

Reply:
At @ZachEddy's recommendation, it looks like it was a problem due to an old version of go.